spirit/comment/models.py

```diff
@@ -47,6 +49,11 @@ class Meta:
         verbose_name = _("comment")
         verbose_name_plural = _("comments")
 
+    def __str__(self):
+        return _('%s by %s at %s') % (
```

Comment:
The named format would be of help when translating `'%(action)s by %(user)s at %(date)s' % {'action': ...}`